wormhole-connect/scripts/ofac/utils.ts

```diff
@@ -0,0 +1,14 @@
+export const getSanctionedAddresses = async (): Promise<string[]> => {
+  const data = await readFileFromUrl(OFAC_SDN_LIST_URL);
+  const ethAddresses = data
+    .match(/Digital Currency Address - [a-zA-Z0-9]+ (0x)?([a-fA-F0-9]{40})/g)
+    ?.map((m) => ensure0x(m.split(' ').at(-1) ?? ''));
+  return [...new Set(ethAddresses)];
+};
+
+const OFAC_SDN_LIST_URL = 'https://www.treasury.gov/ofac/downloads/sdn.csv';
+
+const ensure0x = (str: string) => (str.startsWith('0x') ? str : `0x${str}`);
```

Comment:
What about addresses that don't start with `0x` (i.e. Solana)?

Reply:
This is a good point, do you think I should create a single list of all sanctioned addresses that we compare against? Or would it be better to create an object/Map of form `chainId => Array<sanctionedAddresses>`

Reply:
I think a single list is fine, it's not that long of a list.  We don't just support EVM, so we should account for other types of addresses also

Reply:
Done